app/mailers/candidate_mailer.rb

```diff
@@ -6,4 +6,17 @@ def submit_application_email(application_form)
               to: application_form.candidate.email_address,
               subject: t('submit_application_success.email.subject'))
   end
+
+  def application_under_consideration(application_form)
+    @reject_by_default_days = TimeLimitCalculator.new(
```

Comment:
We can get this from the database:

https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training/blob/c2d11a372d3f5a9d6f4994e0a87d265be0f3d90d/db/schema.rb#L31

Reply:
Good point. The question then, of course, is "which application_choice's reject_by_default_days is the right one?" This is a property of the batch as a whole. Shall we leave it at `application_form.application_choices.first.reject_by_default_days` for now?